content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -141,6 +141,80 @@ limit.
 On Linux, resource allocations for Pod level control groups (cgroups) are based on the effective Pod
 request and limit, the same as the scheduler.
 
+### Adopting sidecar containers
+
+This section is relevant for people adopting sidecar containers for their workloads.
+
+The sidecar containers feature is in beta state starting Kubernetes version 1.29 and is enabled by default.
+Some clusters may have this feature disabled or have software installed that is incompatible with the feature.
+Here are the considerations and troubleshooting steps that one can take while adopting sidecar containers for their workload.
+
+**1. Ensure the feature gate is enabled.**
+
+As a very first step, make sure that both - API server and Nodes were upgraded to the version 1.29+.
+The feature will break on clusters where Nodes are running earlier versions where it is not enabled.
```

Comment:
Is it okay to run a 1.28 version Node with the feature (still Alpha) enabled?

Reply:
added an alert below